legate/cpp/legate_mapping.cpp

```diff
@@ -81,15 +81,11 @@ Legion::Logger log_legate_kvikio(library_name);
 void registration_callback()
 {
   legate::ResourceConfig config;
-  config.max_mappers = 1;
-  config.max_tasks   = OP_NUM_TASK_IDS;
-
-  legate::LibraryContext context(library_name, config);
+  config.max_tasks = OP_NUM_TASK_IDS;
```

Comment:
```c++
legate::ResourceConfig config = {.max_tasks = OP_NUM_TASK_IDS};
```
?
Or are you worried about the struct gaining a constructor somewhere down the line?

Reply:
fixed